.github/workflows/app-test.yml

```diff
@@ -42,22 +47,25 @@ jobs:
         RUNTIME_AUTH: ${{secrets.RUNTIME_AUTH}}
         CUSTOMER_PROFILE_API_TENANT_ID: ${{secrets.CUSTOMER_PROFILE_API_TENANT_ID}}
       run: npm run all
-    - name: Log to file
+    - name: Log to File
+      if: ${{ always() && (github.event_name == 'schedule') }}
       run: |
         git config user.name github-actions
         git config user.email github-actions@github.com
         echo "- $(date) Node ${{ matrix.node }} ${{ job.status == 'success' && '🎉 success' ||  job.status }}" >> logs/run.md
         git add logs/run.md
         git commit -m "generated"
         git push
+    - name: Upload Coverage
+      if: ${{ success() && (github.event_name == 'pull_request' || github.event_name == 'push') }}
+      uses: codecov/codecov-action@v3.1.1
+      with:
+        name: ${{ runner.os }} node.js ${{ matrix.node-version }}
```

Comment:
`matrix.node-version` should be `matrix.node` (either that or change the var name on line 18)

Reply:
Ah... After all this time looking at it too. Fixing

Reply:
Okay fixed